components/base/kinematicbase/differentialDrive.go

```diff
@@ -69,7 +70,7 @@ func wrapWithDifferentialDriveKinematics(
 	}
 	if boundingSphere == nil || err != nil {
 		logger.CWarn(ctx, "base %s not configured with a geometry, will be considered a point mass for collision detection purposes.")
-		boundingSphere = spatialmath.NewPoint(r3.Vector{}, b.Name().Name)
+		boundingSphere = spatialmath.NewPoint(r3.Vector{}, b.Name().ShortName())
```

Comment:
Things dealing with the frame system need the short name, otherwise you probably want the full name.  This is neither of them, but got changed when I did a find/replace I think. 

I tend to think the short name is more appropriate for a geometry label but I can change back

Reply:
We should *always* be using `ShortName`.

The `ShortName()` function will 1) return `Name`, if the resource is local, or 2) return `Name` prepended with with the Remote name and a colon to differentiate it from a locally-named resource with the same name.

It should not be possible to have two resources in a single RDK instance with identical `ShortName`s. It is absolutely possible to have two resources with the same `Name`.